config/main.py

```diff
@@ -1289,6 +1289,31 @@ def hostname(new_hostname):
         raise
     click.echo("Please note loaded setting will be lost after system reboot. To preserve setting, run `config save`.")
 
+#
+# 'synchronous_mode' command ('config synchronous_mode ...')
+#
+@config.command('synchronous_mode')
+@click.argument('sync_mode', metavar='<enable|disable>', required=True)
+def synchronous_mode(sync_mode):
+    """ Enable or disable synchronous mode between orchagent and syncd \n
+        swss restart required to apply the configuration \n
+        Options to restart swss and apply the configuration: \n
+            1. Keep and apply configuration after config reload \n
+                config save -y \n
+                config reload -y \n
+            2. Discard configuration after config reload \n
```

Comment:
>Discard configuration after config reload [](start = 15, length = 41)

This line has nothing to do with next line.

Reply:
Removed the irrelevant line and only kept the commands to restart swss.